monadic-rx/shared/src/main/scala/mhtml/rx.scala

```diff
@@ -154,6 +154,14 @@ case class RxImpureOps[+A](self: Rx[A]) extends AnyVal {
    * If you use this in your code, you are probably doing in wrong.
    */
   def foreach(effect: A => Unit): Cancelable = Rx.run(self)(effect)
+
+  /**
+    * Memoizes this `Rx` using an internal `Var`. This is primarily
+    * useful for optimizing an Rx graph, so that this `Rx` is only
```

Comment:
, so that values generated by this `Rx` are computed only once and shared between all executions.